omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/ManageAppsPageController.java

```diff
@@ -31,27 +31,33 @@
 
 @Controller
 public class ManageAppsPageController {
+	public List<String> appsThatCannotBeStopped = new ArrayList<String>();
+	public ManageAppsPageController(){
+		appsThatCannotBeStopped.add("coreapps.systemAdministrationApp");
+	}
```

Comment:
just FYI you could have done:

    List<String> appsThatCannotBeStopped = Arrays.asList("coreapps.systemAdministrationApp");